common/src/main/java/com/wynntils/gui/render/Texture.java

```diff
@@ -36,6 +36,10 @@ public enum Texture {
     QUEST_BOOK_SEARCH("quest_book_search.png", 133, 23),
     QUEST_BOOK_TITLE("quest_book_title.png", 168, 33),
     RELOAD_BUTTON("reload_button.png", 40, 20),
+    SORT_ALPHABETICALLY("sort_alphabetical.png", 23, 46),
+    SORT_DISTANCE("sort_distance.png", 23, 46),
+    SORT_LEVEL("sort_level.png", 23, 46),
+    SORT_NONE("sort_none.png", 23, 46),
```

Comment:
Did you do "none" anyway? I don't think it makes any point for the user; it was just meant as a way to bypass costly sorting for situations where the sorting did not matter.

Reply:
We need a sort order which is like default quest book. I figured none can be like that (legacy has this too)

Reply:
It might even be better to remove it, and just keep the `getQuests` for a "raw" access to unsorted quests. (In my branch I did actually remove this, or rather let getQuests always take a sort order).

Reply:
Default sort order is more or less level order, is it not?

Reply:
And in any case, we always to a "higher level" sorting of placing started and "can start" quests before cannot start and finished, right? That never changes..?

Reply:
Yeah so any order + started, can start, cannot start, finished + (level order if multiple are matching, for same level too, name).

Reply:
Can you do that? If so, can we merge and you can fix afterwards? Or will we rename "none" to default or smth?

Reply:
Ok, I'll fix that.